clientconn.go

```diff
@@ -1223,6 +1260,11 @@ func (ac *addrConn) updateConnectivityState(s connectivity.State, lastErr error)
 	if ac.state == s {
 		return
 	}
+	locality, backendService := fetchLabels(ac)
```

Comment:
Instead of fetching the labels everytime the connectivity state changes, would it make sense to fetch them when the `addrConn` is created as part of handling `NewSubConn` and updating the labels when `UpdateAddresses` is invoked?